### PR TITLE
Jupyter{Book} Enhancement 

### PR DESCRIPTION
**Pull request by lohithmunakala**
This enhancement is based on the https://github.com/ioos/notebooks_demos/issues/417 (issue). 

A new footer section has been added and the logo has been made bigger.

### The homepage looks like this now
image

### CODE REVIEW

**Comment by MathewBiddle** [comment]
Just pulled this over and built locally. Looks like the colab link has something wrong.
If I try to launch Colab, as shown in the screen shot below, I get a link to the local notebook not a colab notebook.
image

Here's the link I get: 
`file:///github/ioos/notebooks_demos/blob/master/jupyterbook/content/Code%20Gallery/data_access_notebooks/2016-10-12-fetching_data.ipynb`

I think _config.yml needs to be updated, https://github.com/lohithmunakala/notebooks_demos/blob/36c6bcb7cb1cb93392d65599fbbf5391110511f2/jupyterbook/_config.yml#L77

File: jupyterbook/_config.yml [hunk 1]
```diff
@@ -40,13 +40,21 @@ parse:
...
 html:
-  favicon                   : "images/IOOS-logo_little.png"  # A path to a favicon image
+  favicon                   : "images/IOOS_logo.png"  # A path to a favicon image
   use_edit_page_button      : false  # Whether to add an "edit this page" button to pages. If `true`, repository information in repository: must be filled in
   use_repository_button     : false  # Whether to add a link to your repository button
   use_issues_button         : true  # Whether to add an "open an issue" button
-  extra_navbar              : Powered by <a href="https://jupyterbook.org">Jupyter Book</a>  # Will be displayed underneath the left navbar.
-  extra_footer              : ""  # Will be displayed underneath the footer.
-  google_analytics_id       : ""  # A GA id that can be used to track book views.
+  extra_navbar              : <a href="https://ioos.us">IOOS</a>   # Will be displayed underneath the left navbar.
```
Comment (MathewBiddle) [inline jupyterbook/_config.yml:47]: +1 I like this addition!

**Comment by MathewBiddle** [comment]
@mwengren see the screen shot in the [first comment](https://github.com/ioos/notebooks_demos/pull/423#issue-685429365). Is that footer sufficient for alignment with ioos.us sites?

IMHO it's not very pretty, but it relays the information you requested. I think just having the ioos.noaa.gov, Facebook, Twitter, and Contact Us links would suffice.

**Comment by mwengren** [comment]
Looks good to me.  We should probably remove the Copyright though as I don't think that's the intention and everything we produce should be public domain.  We may have that elsewhere in our ioos.github.io sites as well and it should be removed there as well if so (separate issue).

**Comment by MathewBiddle** [comment]
@lohithmunakala can we remove the copyright?

**Comment by lohithmunakala** [comment]
> Just pulled this over and built locally. Looks like the colab link has something wrong.
> If I try to launch Colab, as shown in the screen shot below, I get a link to the local notebook not a colab notebook.
> image
> 
> Here's the link I get:
> `file:///github/ioos/notebooks_demos/blob/master/jupyterbook/content/Code%20Gallery/data_access_notebooks/2016-10-12-fetching_data.ipynb`
> 
> I think _config.yml needs to be updated, https://github.com/lohithmunakala/notebooks_demos/blob/36c6bcb7cb1cb93392d65599fbbf5391110511f2/jupyterbook/_config.yml#L77

I didn't add the Colab link in the `_config.yml` as weren't sure - initially - if we were going to need it. Just added it in this - https://github.com/ioos/notebooks_demos/pull/423/commits/15186f06d48483e3b32eef400c59bd3e931c5e3a - commit.  The option will now direct you to Colab.

**Comment by lohithmunakala** [comment]
> @lohithmunakala can we remove the copyright?

There isn't a way we can remove the term "Copyright". This is what gets displayed when we remove anything from the copyright string in the `config` file. 

image

Or we can do something like this, which is the default

image

Which would we better?

**Comment by MathewBiddle** [comment]
There might be a workaround, see this https://github.com/executablebooks/jupyter-book/issues/582#issuecomment-629371516

**Comment by MathewBiddle** [comment]
@mwengren is it a deal breaker if we can't remove the copyright? Looks like it could get messy.

**Comment by lohithmunakala** [comment]
> There might be a workaround, see this [executablebooks/jupyter-book#582 (comment)](https://github.com/executablebooks/jupyter-book/issues/582#issuecomment-629371516)

I did check this out, but as the static files that are generated are after we run the notebook, it would mess with the automatic publication of the website.  Because we use GitHub Actions to build the notebook, this would get overwritten every time we make a change to the repo - triggering the GA to create a new set of static files. 

We would probably need to automate editing out each of the CSS files and then publish the web page.

**Comment by mwengren** [comment]
Ok, if it's too cumbersome to remove the Copyright, then we can leave as is.  It seems like the JupyterBook library should allow the option to customize that wording though, might be a good PR to submit to them if you know how it could be done.

File: jupyterbook/_config.yml [hunk 2]
```diff
@@ -40,13 +40,21 @@ parse:
 #######################################################################################
 # HTML-specific settings
 html:
-  favicon                   : "images/IOOS-logo_little.png"  # A path to a favicon image
+  favicon                   : "images/IOOS_logo.png"  # A path to a favicon image
   use_edit_page_button      : false  # Whether to add an "edit this page" button to pages. If `true`, repository information in repository: must be filled in
   use_repository_button     : false  # Whether to add a link to your repository button
```
Comment (MathewBiddle) [inline jupyterbook/_config.yml:45]: Let's turn these on too.
image

File: jupyterbook/_config.yml [hunk 3]
```diff
@@ -40,13 +40,21 @@ parse:
...
   use_issues_button         : true  # Whether to add an "open an issue" button
```
Comment (MathewBiddle) [inline jupyterbook/_config.yml:46]: Any way we can use the [existing template](https://github.com/ioos/notebooks_demos/blob/master/.github/ISSUE_TEMPLATE.md)?

**Review by MathewBiddle — COMMENTED** [review]
I would like to have the GH edit and repo buttons available and maybe use our existing ticket template for the issues? Or, leave the defaults here and link somewhere to that ticket template.

File: jupyterbook/_config.yml (hunk 2 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml:45]: Sure!

File: jupyterbook/_config.yml (hunk 3 above)
Comment (lohithmunakala) [reply jupyterbook/_config.yml:46]: I'll check.

**Comment by MathewBiddle** [comment]
@ocefpaf once those changes are made, I'm okay with merging. Unless you want to compile #413 in here and update the various references to `Notebook gallery` or `Data Demo Center` to `IOOS Code Lab`.

**Comment by MathewBiddle** [comment]
@lohithmunakala can we rename this markdown to NANOOS.md

https://github.com/lohithmunakala/notebooks_demos/blob/jupyterbook_enhancement/jupyterbook/content/Video_Demo_Markdown/noos.md

**Comment by lohithmunakala** [comment]
The header for the file inside? Or do we want the name of the file to be changed too.

**Comment by MathewBiddle** [comment]
both

**Comment by MathewBiddle** [comment]
Looks good! I'll merge.

**Pull request merged**